scripts/docker/hub/Dockerfile

```diff
@@ -10,27 +10,23 @@ RUN apt update && apt install -y --no-install-recommends openssl libudev-dev fil
 # show backtraces
 ENV RUST_BACKTRACE 1
 
-#cleanup Docker image
-RUN apt autoremove -y
-RUN apt clean -y
-RUN rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
+# cleanup Docker image
+RUN apt autoremove -y \
+  && apt clean -y \
+  && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
 
 RUN groupadd -g 1000 parity \
-  && useradd -m -u 1000 -g parity -s /bin/sh parity
+  && useradd -m -u 1000 -g parity -s /bin/bash parity
```

Comment:
Not sure whether changing `sh` to `bash` would impose any issues, but maybe this change is just not needed?